src/Sfa.Das.ApprenticeshipInfoService.Infrastructure/Mapping/ProviderMapping.cs

```diff
@@ -127,6 +127,36 @@ public ApprenticeshipDetails MapToProvider(FrameworkProviderSearchResultsItem it
             };
         }
 
+        public ProviderSearchResponseItem MapToProviderSearchItem(ProviderSearchResultsItem provider)
+        {
+            var response = new ProviderSearchResponseItem
```

Comment:
Would be a nice to have if this was something similar to a Paged<ProviderSearchResponseItem> where on the Paged we could give the consumer the total number of records and pages, and page that they have got